src/main/kotlin/no/skatteetaten/aurora/boober/feature/ToxiproxySidecarFeature.kt

```diff
@@ -183,16 +262,51 @@ class ToxiproxySidecarFeature(
             args = listOf("-config", "$configPath/toxiproxy/config.json")
```

Comment:
Vi må legge til "-host=0.0.0.0" for at ToxiProxy admin-api skal ta i mot forbindelser utenfor localhost.
https://github.com/Shopify/toxiproxy/issues/263